packages/kit/src/core/build/build_client.js

```diff
@@ -29,6 +29,8 @@ export async function build_client({
 	output_dir,
 	client_entry_file
 }) {
+	if (!process.env.VITE_APP_VERSION) process.env.VITE_APP_VERSION = new Date().toISOString();
```

Comment:
We need a way to specify the version string via config. It should fall back to a timestamp (`Date.now()` is fine, I think — no need for an ISO string) but I've seen cases where the same app version is built multiple times, and the builds disagree. In those cases you need to be able to specify a commit hash or a `package.json` version or whatever.

Reply:
I thought that using VITE_APP_VERSION environment variable makes using a commit hash already very easy, but I agree that having all in one place in the config is also good. I propose that we set VITE_APP_VERSION to the value configured in the config file, but if there is nothing, it can also be set via environment. the fallback to `Date.now()`is later anyway.